# SPDX-FileCopyrightText: Huawei Inc.
#
# SPDX-License-Identifier: Apache-2.0

Use proper system include direction for input-event-codes.h header.

Apply to repo third_party/libinput

Signed-off-by: Esben Haabendal <esben.haabendal@huawei.com>
Upstream-Status: Pending

diff --git a/include/linux/linux/input.h b/include/linux/linux/input.h
index 857480785ec9..94a04cdf0703 100644
--- a/include/linux/linux/input.h
+++ b/include/linux/linux/input.h
@@ -15,7 +15,7 @@
 #include <sys/types.h>
 #include <linux/types.h>
 
-#include "input-event-codes.h"
+#include <linux/input-event-codes.h>
 
 /*
  * The event structure itself
